widget.lua

```diff
@@ -1012,6 +1038,7 @@ function Widgets.slider_fill:reset()
     self._rangeR=self.axis[2]
     self._rangeWidth=self._rangeR-self._rangeL
     self._pos=self._rangeL
+    self._pos0=self._rangeL
```

Comment:
why

Reply:
Idk, adding slider_fill at my side cause crash so I add that line for fix
Also I guess pos0 is the original value of pos so...